memote/support/helpers.py

```diff
@@ -367,6 +367,31 @@ def find_exchange_rxns(model):
             if extracellular in rxn.get_compartments()]
 
 
+def find_tra_bou_bio_reactions(model, biomass=None):
+    """
+    Return the set of all tranport, boundary, and biomass reactions in a model.
+
+    These reactions are incorporated in models simply for to allow
+    metabolites into the correct compartments for vital system reactions to
+    occur. As such, many tests do not look at these reactions.
+
+    Parameters
+    ----------
+    model : cobra.Model
+        A cobrapy metabolic model
+
+    biomass : list or cobra.Reaction, optional
+        A list of cobrapy biomass reactions
+
+    """
+    # exchanges in this case also refer to sink and demand reactions
+    exchanges = set(model.exchanges)
+    transporters = set(find_transport_reactions(model))
+    if biomass is None:
```

Comment:
Please see my comment above, I don't fully understand why this is necessary.

Reply:
Copying here to explain why I did this:

>The reason I passed in the argument was to remove unnecessary/redundant calculations. Both this function and the function it called (find_tra_bou_bio_reactions) call the same function (find_biomass_reaction). So in order to prevent the function being called twice, I added in an optional, second parameter so the biomass reaction would only be calculated once.

P.S. `find_tra_bou_bio_reactions` has been renamed to `find_interchange_biomass_reactions`